external/kinesis-asl/src/main/scala/org/apache/spark/streaming/kinesis/SerializableCredentialsProvider.scala

```diff
@@ -83,3 +84,146 @@ private[kinesis] final case class STSCredentialsProvider(
     }
   }
 }
+
+@InterfaceStability.Stable
```

Comment:
For sure. Honestly I just cribbed these annotations from ```SparkSession.Builder``` so I appreciate you letting me know what the proper convention is.